firestore-send-email/extension.yaml

```diff
@@ -104,11 +104,12 @@ params:
   - param: DEFAULT_FROM
     type: string
     label: Default FROM address
-    validationRegex: ^\S+@\S+\.\S+$
+    validationRegex: ^(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$|^.*<(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})>$
     validationErrorMessage: Must be a valid email address
     required: true
     description: >-
       The email address to use as the sender's address (if it's not specified in the added email document).
+    example: foobar <foobar@example.com>  
```

Comment:
"foobar" is not obviously someone's name, how about something like: `Friendly Firebaser <foobar@example.com>` ?

Reply:
No problem, I wasn't sure what would be reasonable.

Reply:
this looks done, yeah?  Resolve?